tests/sentry/api/endpoints/test_event_ai_suggested_fix.py

```diff
@@ -104,6 +104,11 @@ def test_consent(client, default_project, test_event, openai_policy):
     assert response.status_code == 403
     assert response.json() == {"restriction": "subprocessor"}
 
+    openai_policy["result"] = "pii_certification_missing"
```

Comment:
@trillville I'm guessing this test is not passing because the user making the request isn't passing the `is_sentry_staff`.  You may need to update the user object before this invocation.

Of course this begs the question of your logic.  If you're not a staff, the current logic falls through an allows the action.

What IS the new openai policy result?  You are setting it to `"pii_certification_missing"` but I don't understand where this comes from.  This looks like the policy_failure, not the policy result.

I honestly think your logic isn't quite right and this test failure is right.

Reply:
Ah, yeah, I think ultimately the fix here is changing the conditional, now that I see the getsentry side of this.